### PR TITLE
Improviments about a11y

### PR DESCRIPTION
**Pull request by brunopulis**
FIxing old issues: 

- Skip Link Navigation
- Correctly <noscript> on document
- Create class to hide skip links
- Create hidden headings to all section
- Wrapping content to <main> tag.

### CODE REVIEW

**Comment by brunopulis** [comment]
@obetomuniz yes i can solve the current code conflit. Regarding the visual changes, no changes were made

File: src/index.html [hunk 1]
```diff
@@ -92,14 +91,6 @@ <h1 class="about-title">Comunidade de Desenvolvedores de Minas Gerais</h1>
                 <p class="about-text">Acreditamos que a troca de conhecimento é a base do crescimento profissional. Queremos unir desenvolvedores para que juntos todos possam contribuir e fortalecer o mercado em Minas Gerais.</p>
```
Comment (obetomuniz) [inline src/index.html]: Why are you changing the current content on this section?

Comment (obetomuniz) [reply src/index.html]: Can you revert it?

Comment (brunopulis) [reply src/index.html]: @obetomuniz Sorry, delete it by mistake.

**Pull request closed without merging**